packages/backend/src/macadam.ts

```diff
@@ -49,6 +50,16 @@ export class MacadamHandler {
           }
           options.imagePath = options.imagePath.replace(/^~\//, `${process.env.HOME}/`);
 
+          // force the correct provider so we're not dependent on .config/containers/containers.conf
+          let provider: 'wsl' | 'hyperv' | 'applehv' | undefined;
+          if (extensionApi.env.isWindows) {
+            provider = (await isWSLEnabled()) ? 'wsl' : 'hyperv';
```

Comment:
For the `hyperv` I think this is a bug on RHEL ext we need to fix, as macadam does not support it yet AFAIK (https://github.com/crc-org/macadam/issues/60)

Reply:
Ok. We don't support Windows at all yet so I was just trying to keep the code in sync with RHEL ext, if there's an update there we'll pick it up later, or we start sharing this code somewhere in the future.